app/src/main/java/pl/org/akai/audioai/screens/LoginScreen.kt

```diff
@@ -1,31 +1,147 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import androidx.navigation.NavController
 import pl.org.akai.audioai.navigation.Screen
 
 @Composable
 fun LoginScreen(navController: NavController) {
 
-    Column (verticalArrangement = Arrangement.Center) {
-        Text("LoginScreen")
-        Spacer(modifier = Modifier.height(8.dp))
-        Button(onClick = {
-            navController.navigate(Screen.AssistantScreen.route)
-        },
-            modifier = Modifier.align(Alignment.End)
-        ) {
-            Text(text = "To AssistantScreen")
+    // Chyba button Login będzie odpowiadał za navi do AssistantScreena
+//    Column(
+//        modifier = Modifier
+//            .padding(20.dp)
+//            .fillMaxHeight(),
+//        verticalArrangement = Arrangement.Bottom,
+//        horizontalAlignment = Alignment.CenterHorizontally
+//    ) {
+//        Spacer(modifier = Modifier.height(8.dp))
+//        Button(
+//            onClick = { navController.navigate(Screen.AssistantScreen.route) },
+//            modifier = Modifier.align(Alignment.End)
+//        ) {
+//            Text(text = "To AssistantScreen")
+//        }
+//    }
+
+    Column(
+        modifier = Modifier
+            .padding(50.dp),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+
+        val context = LocalContext.current
+        val username = remember { mutableStateOf("") }
+        val password = remember { mutableStateOf("") }
+
+        Spacer(modifier = Modifier.height(100.dp))
+        Text(text = "Login", style = TextStyle(fontSize = 40.sp))
+
+        Spacer(modifier = Modifier.height(20.dp))
+        TextField(
+            label = { Text(text = "Username") },
+            value = username.value,
+            onValueChange = { username.value = it })
+
+        Spacer(modifier = Modifier.height(20.dp))
+        TextField(
+            label = { Text(text = "Password") },
+            value = password.value,
+            visualTransformation = PasswordVisualTransformation(),
+            keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Password),
+            onValueChange = { password.value = it })
+
+        Spacer(modifier = Modifier.height(20.dp))
+        Box(modifier = Modifier.padding(40.dp, 0.dp, 40.dp, 0.dp)) {
```

Comment:
suggestion: I think this box is unnecessary, you can move padding to the button's modifier